Source/Scene/ModelExperimental/ModelExperimentalSceneGraph.js

```diff
@@ -118,7 +118,7 @@ export default function ModelExperimentalSceneGraph(options) {
   this.modelPipelineStages = [];
 
   this._boundingSphere = undefined;
-  this._computedModelMatrix = Matrix4.clone(this._model.modelMatrix);
+  this._computedModelMatrix = new Matrix4();
```

Comment:
I would initialize this to `Matrix4.IDENTITY`, the default is all 0s

Reply:
oh if this gets updated in place, then you'll probably have to do `Matrix4.clone(Matrix4.IDENTITY)` since `IDENTITY` is frozen.